fix: support _icontains/_nicontains on json fields via CAST to text (#26928)
directus/directus: .changeset/metal-zebras-wish.md

---
'@directus/utils': patch
'@directus/api': patch
---

> fix: support \_icontains/\_nicontains filter on json fields via text cast

Reviewer: This was not a bug but a feature
```suggestion
Added support for contain queries on json fields
```
